great-idea/css/index.css

```diff
@@ -63,4 +63,112 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+body {
+	margin: 60px 100px;
+  }
+  
+  header nav a {
+	color: gray;
+	margin: 35px 3%;
+	text-decoration: none;
+  }
+  
+  header nav {
+	width: 80%;
+  }
+  
+  header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-bottom: 80px
+  }
+  
+  h1 {
+	font-size: 80px;
+	padding: 20px;
+  }
+  
+  article {
+	text-align: center;
+	display: flex;
+	align-content: space-between;
+  }
+  
+  button:hover {
```

Comment:
Great job animating the buttons. Take a look at different CSS properties like `transition` and `transform`